main/src/ca/uwaterloo/flix/api/lsp/CompletionItem.scala

```diff
@@ -48,18 +48,18 @@ object CompletionItem {
   *                         will be ignored.
   */
 case class CompletionItem(
-  label: String,
```

Comment:
I don't like this formatting. Should we add a rule?

Reply:
But I think it is standard. Its like that elsewhere.